cmd/ddev/cmd/dotenv-get.go

```diff
@@ -69,7 +72,7 @@ ddev dotenv get .ddev/.env.redis --redis-tag`,
 		}
 
 		if !strings.HasPrefix(flag, "--") {
-			util.Failed("The flag must be in a long format.")
+			util.Failed("The flag must be in long format, but received %s", flag)
```

Comment:
Should this be a fail or a warning?

Reply:
`ddev dotenv get` accepts only one flag, should fail here.